### DISCUSSION TITLE
Electron and native module

### DISCUSSION THREAD

**Discussion by loriswave** [vitest-dev/vitest#2142] [Q&A]
Hi,
I'll try to make test on my enviorment and I found a issue. 
My enviorment is composed by vite\vitest, Electron and Better-Sqlite3. 
Sqlite is native module so need to be compiled. For using Sqlite with Electron you need to compile with Electron's Node version. But this version of Node is not the same installed globaly. This mean that the test of SQLite fail. There is a way to force ELECTRON_RUN_AS_NODE and then run the test using this. So in a CMD (windows) you can:
set ELECTRON_RUN_AS_NODE=true
in order to set the enviorment and then run: 
node_modules\.bin\electron node_modules\vitest\vitest.mjs --config config\vitest.config.js

the problem is that the output is:
 
![Immagine](https://user-images.githubusercontent.com/60928061/195395169-60290c26-4959-42c1-9662-d9ed3c0fd810.jpg)

if you run the command:
node node_modules\vitest\vitest.mjs --config config\vitest.config.js
The output is good. Any idea to how to resolve it?

**Comment by linonetwo** [comment]
Use powershell, or enable unicode support in win11 timezone and language setting.
